lwgenerate/internal/helpers.go

```diff
@@ -0,0 +1,37 @@
+package internal
```

Comment:
Can you rename this file https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common
If these functions must be exposed here then call the file something related. But I have a feeling these could be moved to the gcp package